### PR TITLE
[ng] src reorg of toh-5 example, and convert Jade to .md

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #713

### CODE REVIEW

File: src/angular/guide/router/3.md [hunk 1]
```diff
@@ -357,7 +357,7 @@ Notice that the URL now ends with a query parameter like this: `.../heroes?id=15
 
 <div class="l-sub-section" markdown="1">
   The router can also encode parameters using the
-  [matrix URL notation](http://www.w3.org/DesignIssues/MatrixURIs.html),
+  [matrix URL notation](http://www.w3.org/DesignIssues/MatrixURIs),
```
Comment (kwalrath) [inline src/angular/guide/router/3.md]: Let's make this an HTTPS URL:

https://www.w3.org/DesignIssues/MatrixURIs

Comment (chalin) [inline src/angular/guide/router/3.md]: How about if I do a sweep of all pages afterwards?

File: src/angular/tutorial/toh-pt5.md [hunk 2]
```diff
@@ -0,0 +1,1099 @@
...
+  <?code-pane "lib/src/hero_detail_component.css"?>
+  <?code-pane "lib/src/dashboard_component.css"?>
+</code-tabs>
+
+### Style the navigation links
+
+The provided CSS makes the navigation links in the `AppComponent` look more like selectable buttons.
+You'll surround those links in `<nav>` tags.
+
+Add an `app_component.css` file to the `lib/src` folder with the following content.
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt5.md]: `lib/src` -> `lib`

File: src/angular/guide/router/3.md (hunk 1 above)
Comment (chalin) [inline src/angular/guide/router/3.md]: Fixed

File: src/angular/tutorial/toh-pt5.md (hunk 2 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt5.md]: Fixed!

**Pull request merged**